### PR TITLE
Date format Part2

### PR DESCRIPTION
**Pull request by charlesabarnes**
## **Description**

I added an option for moment and I pass in the format option through a couple of places so it actually get used in the component.   

#### **Verify that...**

- [x] Any related demos where added and `npm start` still works
- [x] New demos work in `Safari`, `Chrome` and `Firefox`
- [x] `npm run lint` passes
- [x] `npm test` passes and code coverage is increased
- [x] `npm run compile` still works

##### **Screenshots**

image

### CODE REVIEW

File: src/platform/elements/date-picker/DatePickerInput.ts [hunk 1]
```diff
@@ -45,20 +47,30 @@ export class NovoDatePickerInputElement implements OnInit, ControlValueAccessor
   @Input() placeholder: string;
   @Input() maskOptions: any;
   @Input() format: string;
+  @Input() textMaskEnabled: boolean;
```
Comment (jgodi) [inline src/platform/elements/date-picker/DatePickerInput.ts]: Can default to true, then you don't need to check for undefined below

Comment (jgodi) [reply src/platform/elements/date-picker/DatePickerInput.ts]: nvm, I see how you are doing it, maybe flip it? `disabledTextMask: boolean = false` then down below itll be `!!field.disableTextMask`

**Comment by jgodi** [comment]
|File (2)|Change (:bangbang: -0.26%)|
|---|---|
|Address.ts|:white_check_mark: +0.00%|
|FormUtils.ts|:white_check_mark: +0.00%|

*Please note, individual file changes won't add up to the overall coverage due to number of lines!*

**All new files must be at 80% coverage and old files must NOT decrease in coverage!**

File: src/platform/elements/date-picker/DatePickerInput.ts (hunk 1 above)
Comment (charlesabarnes) [reply src/platform/elements/date-picker/DatePickerInput.ts]: You were right.  I wanted it to just default to true. I just set the default.  It works just fine.

**Pull request merged**